static/app/components/demo/demoHeader.tsx

```diff
@@ -36,7 +36,7 @@ export default function DemoHeader() {
       <RequestDemoBtn
         priority="form"
         onClick={() =>
-          trackAdvancedAnalyticsEvent('growth.demo_click_request_demo', {
+          trackUserOnlyAnalyticsEvents('growth.demo_click_request_demo', {
             organization: null,
```

Comment:
you can delete the organization here now right?

Reply:
Ah I see, the type is still required.

Probably can modify the makeAnalyticsFunction so that it doesn't require an org if that type is null in the template

Reply:
You mean as opposed to making `organization: null`?